prow/plugins/assign/assign.go

```diff
@@ -131,15 +155,56 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	if ok, merr := gc.IsMember(ae.org, ae.login); merr == nil && !ok {
```

Comment:
Wait, we want anyone to be able to /assign, not just org members. However we only want the /assign at-target to be an org member

What we should do here is verify that A, B, C are org members for the comment `/assign at-A at-B at-C`.

Who makes the `/assign at-A at-B at-C` comment is totally irrelevant